docs/reference/remap/functions/parse_apache_log.cue

```diff
@@ -0,0 +1,112 @@
+package metadata
+
+remap: functions: parse_apache_log: {
+	category:    "Parse"
+	description: """
+		Parses apache access and error log lines. Lines can be either in [`common`](\(urls.apache_common)) format,
+		[`combined`](\(urls.apache_combined)) format, or default
+		[`error`](\(urls.apache_error)) format.
+		"""
+	notices: [
+		"""
+			Missing information in the log message may be indicated by `-`. These fields will not be present in the result.
+			""",
```

Comment:
Seems like this file could use a `cue fmt`.

Reply:
Sometimes it feels like `cue fmt` just goes on vacation..

Reply:
No, it seems that really is how `cue fmt` likes it..